packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md

```diff
@@ -37,6 +37,9 @@ TODO: Remove this section if there are not any general updates.
 
 * Added a tooltip to describe the exception mode drop-down. -
 [#8849](https://github.com/flutter/devtools/pull/8849)
+* Either the 'pause' or the 'resume' buttons is displayed,
+  never both. -
+  [#4924](https://github.com/flutter/devtools/issues/4924)
```

Comment:
Nit: Please link the PR instead of the issue that this fixes. 
Uber nit on verbiage: entries should be phrased in the past tense (e.g. "Added XYZ" instead of "Add XYZ").